lib/services/azurepostgresqldb/cmd-bind.js

```diff
@@ -0,0 +1,74 @@
+/* jshint camelcase: false */
+
+'use strict';
+
+var async = require('async');
+var common = require('../../common');
+var Config = require('./service');
+var log = common.getLogger(Config.name);
+
+var postgresqldbBind = function (params) {
+    var provisioningResult = params.provisioning_result;
+
+    this.bind = function (postgresqldbOperations, next) {
+        var databaseLogin, databaseLoginPassword;
+        async.waterfall([
+            function (callback) {
+                // Generate the login/password
+                databaseLogin = common.generateName();
+                databaseLoginPassword = common.generateStrongPassword();
+                // Create the login
+                postgresqldbOperations.createDatabaseLogin(
+                    provisioningResult.fullyQualifiedDomainName,
+                    provisioningResult.postgresqlServerName,
+                    provisioningResult.postgresqlDatabaseName,
+                    provisioningResult.administratorLogin,
```

Comment:
Per the PR https://github.com/Azure/meta-azure-service-broker/pull/211 we recently made, SQLDB module switched to get admin login from the account pool (broker manifest). Could you change this to keep aligned with SQLDB?

Reply:
There is currently no support for Postgres pre-provisioned server pools... Do you suggest implementing this pooling feature, similar to SQLDB?

Reply:
Oh sorry, my bad, Yeah, the server pool is not implemented yet.